com.civicrm.hrjobroles/scss/hrjobroles.scss

```diff
@@ -1,124 +1,19 @@
-  // bootstrap/normalize and bootstrap/scaffolding customized to enable
-  // the narrowed scope of the styles for this CiviCRM module only.
-  // ==========================================================================
-  @import "bootstrap-sprockets";
-
-  @import "bootstrap-compass";
-  @import "civihr/bootstrap-variables";
-
-  @import "bootstrap/variables";
-  @import "bootstrap/mixins";
-
-  // Reset and dependencies
-  @import "civihr/bootstrap/normalize-custom";
-  @include normalize;
-  @import "bootstrap/print";
-  @import "civihr/bootstrap/glyphicons-custom";
-  @import "civihr/modules/fontawesome";
-
-  // Core CSS
-  @import "civihr/bootstrap/scaffolding-custom";
-  @include scaffolding;
-  @import "bootstrap/type";
-  @import "bootstrap/code";
-  @import "bootstrap/grid";
-  @import "bootstrap/tables";
-  @import "bootstrap/forms";
-  @import "bootstrap/buttons";
-
-  // Components
-  @import "bootstrap/component-animations";
-  @import "bootstrap/dropdowns";
-  @import "bootstrap/button-groups";
-  @import "bootstrap/input-groups";
-  @import "bootstrap/navs";
-  @import "bootstrap/navbar";
-  @import "bootstrap/breadcrumbs";
-  @import "bootstrap/pagination";
-  @import "bootstrap/pager";
-  @import "bootstrap/labels";
-  @import "bootstrap/badges";
-  @import "bootstrap/jumbotron";
-  @import "bootstrap/thumbnails";
-  @import "bootstrap/alerts";
-  @import "bootstrap/progress-bars";
-  @import "bootstrap/media";
-  @import "bootstrap/list-group";
-  @import "bootstrap/panels";
-  @import "bootstrap/responsive-embed";
-  @import "bootstrap/wells";
-  @import "bootstrap/close";
-
-  // Components w/ JavaScript
-  @import "civihr/bootstrap/modals-custom";
-  @import "bootstrap/tooltip";
-  @import "bootstrap/popovers";
-  @import "bootstrap/carousel";
-
-  // Utility classes
-  @import "bootstrap/utilities";
-  @import "bootstrap/responsive-utilities";
-
-  // CiviHR Theme
-  // ==========================================================================
-  @import "civihr/modules/variables";
-  @import "civihr/modules/print";
-  @import "civihr/modules/fonts";
-  @import "civihr/modules/angular";
-  @import "civihr/modules/utils";
-  @import "civihr/modules/textangular";
-  @import "civihr/modules/mixins";
-  @import "civihr/modules/animate";
-
-  @import "civihr/partials/typography";
-  @import "civihr/partials/global";
-  @include global;
-  @import "civihr/partials/badge";
-  @import "civihr/partials/breadcrumb";
-  @import "civihr/partials/buttons";
-  @import "civihr/partials/collapse";
-  @import "civihr/partials/datepicker";
-  @import "civihr/partials/dropdowns";
-  @import "civihr/partials/forms";
-  @import "civihr/partials/iframe";
-  @import "civihr/partials/modals";
-  @import "civihr/partials/page-contact";
-  @import "civihr/partials/pagination";
-  @import "civihr/partials/panels";
-  @import "civihr/partials/tables";
-  @import "civihr/partials/tooltip";
-  @import "civihr/partials/wells";
+  &__list {
```

Comment:
well this css is so minimal that i wanted to keep things simple, we're talking about 19 lines of code here. If for some reason things get more complicated we can still split it in multiple files